controllers/registrationservice/registrationservice_controller_test.go

```diff
@@ -27,8 +28,8 @@ import (
 func TestReconcileRegistrationService(t *testing.T) {
 	// given
 	s := scheme.Scheme
-	err := toolchainv1alpha1.AddToScheme(s)
-	require.NoError(t, err)
+	require.NoError(t, apis.AddToScheme(s))
+	require.NoError(t, tmplv1.Install(s))
```

Comment:
`tmplv1.Install` is already part of the `apis.AddToScheme()` func, AFAICS

Reply:
good point - removed in f5ae3d9
this is a leftover from the time when I was dealing with a lot of errors of missing template api in the scheme